app/src/main/java/net/squanchy/imageloader/ImageLoaderModule.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.imageloader;
+
+import android.content.Context;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+class ImageLoaderModule {
+
+    private final Context context;
+
+    ImageLoaderModule(Context context) {
+        this.context = context;
+    }
+
+    @Provides
+    ImageLoader imageLoader() {
```

Comment:
822c089d73540ff08fe4bbf83c43c9a201b15776

Fun fact `Glide.with(context)` returns `RequestManager` not `Glide` for some reason